lib/workers/repository/update/branch/index.spec.ts

```diff
@@ -959,28 +1010,35 @@ describe('workers/repository/update/branch/index', () => {
     });
 
     it('branch pr no rebase (dry run)', async () => {
-      git.branchExists.mockReturnValue(true);
-      jest.spyOn(prWorker, 'updatePrDebugData').mockReturnValueOnce({
-        updatedInVer: '1.0.3',
-        createdInVer: '1.0.2',
-      });
-      platform.getBranchPr.mockResolvedValueOnce({
+      const pr = partial<Pr>({
         state: PrState.Open,
-      } as Pr);
+        number: 1,
+      });
+      git.branchExists.mockReturnValue(true);
```

Comment:
```suggestion
      git.branchExists.mockReturnValueOnce(true);
```
shouldn't this only be once?

Reply:
there are 53 instances where we have this in the same file.
where 49 of these can changed as above, i guess it might be better off as a separate PR?
![image](https://user-images.githubusercontent.com/97394622/201924087-15748c99-8ef6-435f-9335-de7a541c7060.png)